src/main/java/io/dropwizard/kafka/KafkaAdminClientFactory.java

```diff
@@ -79,6 +82,11 @@
     @Valid
     @JsonProperty
     protected SecurityFactory security;
+    @JsonProperty
+    protected boolean createTopics;
```

Comment:
Maybe rename this `topicCreationEnabled`? I generally like having the `enabled` word present in these type of configs. Also, I know by default this boolean will be `false`, but maybe making that explicit in code would be a little nicer. `protected boolean createTopics = false;`